src/clusterfuzz/_internal/bot/tasks/update_task.py

```diff
@@ -248,7 +248,12 @@ def update_source_code():
                      'version.' % absolute_filepath)
 
     try:
-      extracted_path = zip_archive.extract(filepath, output_directory)
+      try:
```

Comment:
It's a bit ugly, but I can't really think of a better way to deal with this issue.

Reply:
I could remove every file first, but I don't know if this has any unintended consequences.

Reply:
Another approach: Could we just chmod/chown every extracted first to ensure they're writable?